Packs/Illumio/Integrations/IllumioCore/README.md

```diff
@@ -1,5 +1,5 @@
 Connects to Illumio Core APIs to perform investigative and restorative actions.
-This integration was integrated and tested with version 1.1.1 of IllumioCore
+This integration was integrated and tested with version 1.1.0 of IllumioCore
```

Comment:
This version is not the Illumio Core version, it's the `illumio` python library version.

Reply:
Yes, I've made the changes.